### PR TITLE
chore: providing basic quince support

### PR DESCRIPTION
**Pull request by jfavellar90**
This PR aims to provide Harmony plugin support for the Quince OpenedX release.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jfavellar90! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/__about__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.1.0"
+__version__ = "17.0.0"
```
Comment (farhaanbukhsh) [inline tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/__about__.py:1]: This is the version for the plugin right? So this should be 0.1.1, no?

**Review by farhaanbukhsh — CHANGES_REQUESTED** [review]
Hey @jfavellar90, we tested this PR.

I have tested the following points with the current deployment. I have tested this by verifying the config files generated by Tutor. 


1. The [ingress service](https://github.com/openedx/openedx-k8s-harmony/blob/main/tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/k8s-services) file I checked under `instances/example/env/k8s/services.yml` and it is following the correct template pattern, I checked the deployment as well

The yaml file was correctly generated and applied.

2.  The  [openedx-common-settings](https://github.com/openedx/openedx-k8s-harmony/blob/main/tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/openedx-common-settings) was tested by looking for `ELASTIC_SEARCH_CONFIG` in instances/example/env/apps/openedx/settings/cms/production.py and instances/example/env/apps/openedx/settings/lms/production.py.  This was perfectly present in the file as:


```python

ELASTIC_SEARCH_INDEX_PREFIX = "example-"
ELASTIC_SEARCH_CONFIG = [{
  "use_ssl": True,
  "host": "harmony-search-cluster.harmony.svc.cluster.local",
  "verify_certs": False,
  "port": 9200,
  "http_auth": "example:password"
}]
```

I also tested this on the shell and got the same config.

File: tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/__about__.py (hunk 1 above)
Comment (bradenmacdonald) [reply tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/__about__.py:1]: To keep things simple, plugins are often versioned to match the Tutor version that they support.

**Review by gabor-boros — APPROVED** [review]
Based on @farhaanbukhsh's review, I think this PR can be approved.

@jfavellar90 Feel free to merge this PR if there is nothing else you wanted to add.

**Comment by gabor-boros** [comment]
FTR: Merging as it is a blocker and already got approved.

**Comment by openedx-webhooks** [comment]
@jfavellar90 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-k8s-harmony&entry.867055334=https://github.com/openedx/openedx-k8s-harmony/pull/63&entry.1484655318=https://github.com/jfavellar90&entry.752974735=2024-03-05+16:52&entry.1917517419=2024-04-22+06:13&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**